screens/popups/transfer_pokemon/transfer.gui_script

```diff
@@ -245,7 +245,7 @@ function init(self)
 	gui.set_text(n, "MOVE TO " .. self.to)
 
 	self.move_button_enabled = true
-	if self.to == "storage" and #storage.list_of_ids_in_inventory() == 1 then
+	if self.to == "PC" and #storage.list_of_ids_in_inventory() == 1 then
```

Comment:
Was this a leftover from a previous time when the `PC` was called `storage` instead?

This is a good reason to use constants. Especially with me considering adding encounters, it would be nice to have, like:
```
PokemonLocation.PC
PokemonLocation.PARTY
PokemonLocation.ENCOUNTER
```
Constants like that would help systems communicate more effectively.

Reply:
Yep, a left over. Would definitely be a good idea to do that. Should do another PR for it. There are a lot of other things that should get the same treatment. Like hash (like `"touch"´) and screen names that we use with monarch.